libraries/AP_Landing/AP_Landing_Slope.cpp

```diff
@@ -409,7 +409,7 @@ void AP_Landing::type_slope_log(void) const
 // @Field: slopeInit: Initial slope to landing point
 // @Field: altO: Rangefinder correction
 // @Field: fh: Height for flare timing.
-    AP::logger().Write("LAND", "TimeUS,stage,f1,f2,slope,slopeInit,altO,fh", "QBBBffff",
+    AP::logger().WriteStreaming("LAND", "TimeUS,stage,f1,f2,slope,slopeInit,altO,fh", "QBBBffff",
```

Comment:
these don't look like streaming msgs?

Reply:
it is and accounts for significant entries during autolands...I could not change the deep stall streaming since its a Block write and could not find an equivalent streaming call for it in the short time....but deep stall is very uncommon....just like the Torqueedo streams would be...